src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -420,21 +420,30 @@ private EmailWrapper generateFeedbackSessionEmailBaseForStudents(
         return email;
     }
 
+    private String generateInstructorFragment(String courseId, String courseName) {
+
+        String courseIdentifier = SanitizationHelper.sanitizeForHtml(courseId) + ", "
+                + SanitizationHelper.sanitizeForHtml(courseName);
+
+        return "<p>The email below has been sent to students of course: "
+                + courseIdentifier
+                + ".<br>" + Const.EOL + "<br>" + Const.EOL
+                + "=== Email message as seen by the students ===</p>" + Const.EOL;
```

Comment:
I had followed the same line breaking philosophy with another preamble that was already existing in the code. However, I will fix it asap.